app/pb_kits/playbook/pb_checkbox/_checkbox.jsx

```diff
@@ -0,0 +1,33 @@
+/* @flow */
+
+import React from 'react'
+
+type CheckboxProps = {
+  children: Array<React.ReactNode> | React.ReactNode,
+  dark?: Boolean,
+  tag: String,
+  label: String,
+}
+
+const Checkbox = ({
+  children,
+  dark=false,
+  tag='label',
+  label,
+}: CheckboxProps) => {
+
+  const Tag = `${tag}`
+
+  return (
+    <Tag
+        className={
+        `pb_checkbox_kit` +
+        (dark === true ? '_dark' : '')
+      }
+    > <input type="checkbox"/>
```

Comment:
Also, I think you might want to allow another prop in this kit called `name` so that devs can set it's value which will be used in any Rails form mappings. This prop is also used in a conventional nature in React apps within Nitro as well. e.g.

```jsx
<input name={name} type="checkbox"/>
```

This new prop would also open the doors to composing a toggle if need be in the future.

Reply:
See https://github.com/powerhome/playbook/pull/239/files for the `Toggle` example I mentioned above